cfbs.json

```
@@ -448,6 +448,20 @@
       ]
     },
     "python-lib": { "alias": "library-for-promise-types-in-python" },
+    "sshd-PermitEmptyPasswords-no": {
```

```suggestion
    "ssh-permit-empty-pass-words-no": {
```

I understand that you think this is more technically correct, but I it provides a better user experience if we are consistent, using lowercase names and dashes.

---

I dont like this.

I dont dislike it enough to argue, please make a PR to the repo with the same changes.

---

or, add aliases instead